src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -46,13 +46,17 @@ object IndexConstants {
   val INDEX_HYBRID_SCAN_DELETE_ENABLED = "spark.hyperspace.index.hybridscan.delete.enabled"
```

Comment:
Should we remove this config now? #184 seems to be closed.

Reply:
And `HyperspaceConf.hybridScanDeleteEnabled` can be implemented based on `INDEX_HYBRID_SCAN_DELETED_RATIO_THRESHOLD` (disabled if it is set to 0, for example)